### PR TITLE
manifest: openthread upmerge up to commit `2ce3d3b`

### PR DESCRIPTION
**Pull request by edmont**
Bring Thread 1.3 changes from NCS 2.0.2.

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ manifest:
       url-base: https://github.com/NordicSemiconductor
     - name: memfault
       url-base: https://github.com/memfault
+    - name: edmont
```
Comment (canisLupus1313) [inline west.yml:34]: :) I think You shouldn't release nrfxlib and openthread from Your fork.

**Review by canisLupus1313 — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

File: applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf [hunk 2]
```diff
@@ -18,3 +18,7 @@ CONFIG_BT_BUF_ACL_RX_SIZE=502
 
 # Workaround FEM issue seen on older Thingy:53 revisions
 CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA=10
+
+# Enable the frame encryption feature in the radio driver, it's required for proper working
```
Comment (greg-fer) [inline applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg.conf:22]:
```suggestion
# Enable the frame encryption feature in the radio driver, as it is required for proper working
```

File: applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg_release.conf [hunk 3]
```diff
@@ -18,3 +18,7 @@ CONFIG_BT_BUF_ACL_RX_SIZE=502
 
 # Workaround FEM issue seen on older Thingy:53 revisions
 CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA=10
+
+# Enable the frame encryption feature in the radio driver, it's required for proper working
```
Comment (greg-fer) [inline applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg_release.conf:22]:
```suggestion
# Enable the frame encryption feature in the radio driver, as it is required for proper working
```

File: doc/nrf/ug_thread_ot_memory.rst [hunk 4]
```diff
@@ -39,44 +39,44 @@ nRF5340 DK RAM and flash memory requirements
 
 The following tables present memory requirements for samples running on the :ref:`nRF5340 DK <gs_programming_board_names>` (:ref:`nrf5340dk_nrf5340 <zephyr:nrf5340dk_nrf5340>`) with the software cryptography support provided by the :ref:`nrfxlib:nrf_oberon_readme` module.
 
-.. table:: nRF5340 multiprotocol Thread 1.2 memory requirements
+.. table:: nRF5340 single protocol Thread 1.3 memory requirements
```
Comment (greg-fer) [inline doc/nrf/ug_thread_ot_memory.rst:42]: Just confirming: you're inverting the order of tables? I wouldn't want this to be some typo :)

File: doc/nrf/ug_thread_supported_features.rst [hunk 5]
```diff
@@ -30,6 +30,14 @@ The OpenThread implementation of the Thread protocol supports all features defin
 
   * Service Registration Protocol (SRP) client
 
+* Features introduced with Thread 1.3.0:
+
+  * Service Registration Protocol (SRP) client
```
Comment (greg-fer) [inline doc/nrf/ug_thread_supported_features.rst:35]: Isn't this a duplicate of lines 29-32?

File: doc/nrf/ug_thread_supported_features.rst [hunk 6]
```diff
@@ -30,6 +30,14 @@ The OpenThread implementation of the Thread protocol supports all features defin
...
+
+In |NCS|, you can choose which version of the Thread protocol to use in your application.
+By default, |NCS| supports Thread 1.3, which includes support for Thread 1.2.
+You can enable and configure any Thread version by using :ref:`dedicated options <thread_ug_thread_specification_options>`.
```
Comment (greg-fer) [inline doc/nrf/ug_thread_supported_features.rst:39]: The section linked only mentions versions 1.1 and 1.2, and by the looks of it, it should also mention 1.3. However, maybe we can remove the list of supported features from there and just point to the list in this file instead.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_thread_ot_memory.rst (hunk 4 above)
Comment (edmont) [reply doc/nrf/ug_thread_ot_memory.rst:42]: Yes, exactly, the order was inverted :)

File: doc/nrf/ug_thread_supported_features.rst (hunk 6 above)
Comment (edmont) [reply doc/nrf/ug_thread_supported_features.rst:35]: I missed to pick some changes in `ug_thread_configuring.rst`. Now it should be better.

**Comment by edmont** [comment]
Removed nrfxlib manifest update since it was already part of main.

**Pull request merged**